pkg/result/filter_test.go

```diff
@@ -184,6 +184,27 @@ func TestFilter(t *testing.T) {
 			Category:   "restricted",
 			Confidence: 1,
 		}
+		license3 = types.DetectedLicense{
+			Name:       "MIT AND GPL-2.0-or-later",
```

Comment:
Let's change this license name, to be sure that we normalize license before check rule. 
```suggestion
			Name:       "mit AND GPL-2.0-or-later",
```